content/courses/onchain-development/intro-to-onchain.md

```diff
@@ -11,133 +11,192 @@ description:
-    seeds `USD` and `AUD` to make a Program Derived Address
-  - Want to store information about the relationship of two users? You can use
-    **both those users' wallet addresses** as seeds to make a PDA to store that
+- The 'key' is the PDA address, determined by `seeds` chosen by you, the
+  developer.
+  - For example, if your program needs to store the USD to AUD exchange rate,
+    you can use the seeds `USD` and `AUD` to create a Program-Derived Address.
+  - To store information about the relationship between two users, you can use
+    both users' wallet addresses as seeds to create a PDA to store that
     information.
-  - Want an account to store Steve's review of Titanic? Your program can use
-    Steve's **wallet address** and the string `titanic` (or maybe the IMDB ID if
-    you prefer) to make a Program Derived Address.
-  - Want some global information for your entire program? You can use a string
-    like `'config'`. Your program's PDAs are unique, so they won't conflict with
-    other programs.
-- The value is the data inside the account at the given address.
-  - The data inside the PDA is determined by you, the programmer.
-
-Key value stores allow your onchain program, and client software, to
-consistently determine the address for a data item because the same seeds will
-always return the same address.
+  - To store Steve's review of Titanic, you can use Steve's **wallet address**
+    and the string `titanic` (or the IMDB ID if you prefer) to create a
+    Program-Derived Address.
+  - To store global information for your entire program, you can use a string
+    like `'config'`. Since your program’s PDAs are unique, they won’t conflict
+    with other programs.
+- The 'value' is the data stored inside the account at the given address,
+  defined by you, the developer.
+
+Key-value stores enable your onchain program and client software to consistently
+determine the address for a data item because the same seeds will always return
+the same address.
+
+#### How PDAs Work
+
+1. **Key**: The PDA address, derived from seeds you choose as a developer.
+2. **Value**: The data stored in the account at that address.
+
+#### Examples
+
+Here's a table illustrating various use cases for PDAs:
+
+| Use Case          | Seeds                          | PDA (Key)       | Value (Data Stored)                        |
+| ----------------- | ------------------------------ | --------------- | ------------------------------------------ |
+| Exchange Rate     | `["USD", "AUD"]`               | Derived address | Current USD to AUD exchange rate           |
+| User Relationship | `[user1_wallet, user2_wallet]` | Derived address | Relationship data (e.g., friends, blocked) |
+| Movie Review      | `[reviewer_wallet, "titanic"]` | Derived address | Review text, rating, timestamp             |
+| Global Config     | `["config"]`                   | Derived address | Program-wide settings                      |
```

Comment:
Not sure if I mentioned it before but I love this table, and will use it in our other work.

Reply:
Thanks! @mikemaccana I appreciate it.